compiledb/__init__.py

```diff
@@ -73,10 +72,10 @@ def check_file(path):
     return [v for k, v in orig.items() if check_file(k)]
 
 
-def generate(infile, outfile, build_dir, exclude_files, verbose, overwrite=False, strict=False):
+def generate(infile, logfile, outfile, build_dir, exclude_files, verbose, overwrite=False, strict=False):
```

Comment:
`infile` is not the input compilation database filename, and `logfile` in the input log file stream